man/beamer_presentation.Rd

```diff
@@ -59,16 +59,22 @@ individual bullet you can precede it with \code{>}. For example:
 \item{dev}{Graphics device to use for figure output (defaults to pdf)}
 
 \item{df_print}{Method to be used for printing data frames. Valid values
-include "default", "kable", "tibble", and "paged". The "default" method
-uses a corresponding S3 method of \code{print}, typically
-\code{print.data.frame}. The "kable" method uses the
-\code{\link[knitr:kable]{knitr::kable}} function. The "tibble" method uses
-the \pkg{tibble} package to print a summary of the data frame. The "paged"
-method creates a paginated HTML table (note that this method is only valid
-for formats that produce HTML). In addition to the named methods you can
-also pass an arbitrary function to be used for printing data frames. You
-can disable the \code{df_print} behavior entirely by setting the option
-\code{rmarkdown.df_print} to \code{FALSE}.}
+  include "default", "kable", "tibble", and "paged". The "default" method
+  uses a corresponding S3 method of \code{print}, typically
+  \code{print.data.frame}. The "kable" method uses the
+  \code{\link[knitr:kable]{knitr::kable}} function. The "tibble" method uses
+  the \pkg{tibble} package to print a summary of the data frame. The "paged"
+  method creates a paginated HTML table (note that this method is only valid
+  for formats that produce HTML).
+
+  In addition to the named methods you can also pass an arbitrary function to
+  be used for printing data frames. If specifying this within a YAML header,
+  arbitrary functions must be prefaced by \samp{!expr}, i.e \samp{df_print:
+  !expr knitr::kable} is equivalent to the method "kable". See
+  \code{\link[yaml:yaml.load]{yaml::yaml.load}} for details.
+
+  You can disable the \code{df_print} behavior entirely by setting the option
+  \code{rmarkdown.df_print} to \code{FALSE}.}
```

Comment:
I don't think `\item{}` supports multiple paragraphs, so you need to make this a single paragraph. Perhaps just add a link to the book instead of explaining all the details here? https://bookdown.org/yihui/rmarkdown/html-document.html#data-frame-printing (this page will be updated after I merge https://github.com/rstudio/rmarkdown-book/pull/77)

Reply:
It outputs like this 
![image](https://user-images.githubusercontent.com/6791940/96687391-18ef2700-1380-11eb-8370-8fc03348f5ac.png)

So it seemed ok to me. 

But I guess you are right, now I've suggest addition in the book I should link to that. I have done these improvments the other way around 😅

Reply:
Done in https://github.com/rstudio/rmarkdown/pull/1930/commits/6961ddd3a8a0e61b2054081a3688f3406574ae85